helm/values.yaml

```diff
@@ -65,11 +65,11 @@ env:
   logPrettyPrintEnabled: false
   responseCompressionEnabled: true
   requestPayloadLimit: 1mb
-  tilingZoomGroups: '0-12,13,14,15,16,17,18,19,20,21,22,23'
+  tilingZoomGroups: ['0-12', '13' '14', '15', '16', '17', '18', '19', '20', '21', '22', '23']
```

Comment:
why is it different from https://github.com/MapColonies/discrete-overseer-creator/pull/29/files#diff-81801117ec01136c8a49bfcd42af8e104f4efad1f2daccda9da9d960eaad5279R96
please implement same solution for both